# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: architecture/loongarch64/package/*/have_stdlib.patch
# Copyright (C) 2004 - 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- binutils-2.18.50.0.8/libiberty/cplus-dem.c.vanilla	2008-07-16 20:29:10.000000000 +0200
+++ binutils-2.18.50.0.8/libiberty/cplus-dem.c	2008-07-16 20:29:27.000000000 +0200
@@ -49,12 +49,7 @@
 #include <string.h>
 #include <stdio.h>
 
-#ifdef HAVE_STDLIB_H
 #include <stdlib.h>
-#else
-void * malloc ();
-void * realloc ();
-#endif
 
 #include <demangle.h>
 #undef CURRENT_DEMANGLING_STYLE
